static/bidder-params/rise.json

```diff
@@ -4,10 +4,10 @@
   "description": "A schema which validates params accepted by the Rise adapter",
   "type": "object",
   "properties": {
-    "publisher_id": {
+    "org": {
       "type": "string",
-      "description": "The publisher ID"
+      "description": "The organization ID"
     }
   },
-  "required": ["publisher_id"]
+  "required": ["org"]
```

Comment:
Please give publishers enough transition period to change publisher_id to org. In the meantime, you could support both. 
```
"oneOf": [
    {
      "required": [
        "publisher_id"
      ]
    },
    {
      "required": [
        "org"
      ]
    }
  ]
  ```